tensorflow/core/util/gpu_device_functions.h

```diff
@@ -751,12 +751,16 @@ __device__ inline double GpuAtomicAdd(double* ptr, double value) {
 }
 #endif
 
-#if __gfx908__ || __gfx90a__
+#if __gfx908__ || __gfx90a__  || __gfx940__
```

Comment:
Replicate on line 955

Reply:
> Replicate on line 955

Thanks @ekuznetsov139 ! I assume 966 also is required as well?